src/test/java/pl/szczesniak/dominik/whattowatch/movies/domain/MoviesToWatchServiceTest.java

```diff
@@ -0,0 +1,156 @@
+package pl.szczesniak.dominik.whattowatch.movies.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.szczesniak.dominik.whattowatch.movies.domain.model.MovieId;
+import pl.szczesniak.dominik.whattowatch.movies.domain.model.exceptions.UserDoesNotExistException;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.UserId;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static pl.szczesniak.dominik.whattowatch.movies.domain.TestMoviesToWatchServiceConfiguration.moviesToWatchService;
+
+class MoviesToWatchServiceTest {
+
+    private InMemoryUserProvider userProvider;
+    private MoviesToWatchService tut;
+
+    @BeforeEach
+    void setUp() {
+        userProvider = new InMemoryUserProvider();
+        tut = moviesToWatchService(userProvider);
+    }
+
+    @Test
+    void user_should_add_movie_to_his_list() {
+        // given
+        final UserId userId = new UserId(1);
+        userProvider.addUser(userId);
+
+        // when
+        tut.addMovieToList("Parasite", userId);
+
+        // then
+        assertThat(tut.getList(userId)).hasSize(1);
+    }
+
+    @Test
+    void shouldnt_add_movie_when_user_doesnt_exist() {
+        // given
+        final UserId userId = new UserId(1);
+
+        // when
+        final Throwable thrown = catchThrowable(() -> tut.addMovieToList("Parasite", userId));
+
+        // then
+        assertThat(thrown).isInstanceOf(UserDoesNotExistException.class);
+    }
+
+    @Test
+    void user_should_be_able_to_add_movies_with_same_title() {
+        // given
+        final UserId userId = new UserId(1);
+        userProvider.addUser(userId);
+
+        // when
+        tut.addMovieToList("Parasite", userId);
+        tut.addMovieToList("Star Wars", userId);
+        tut.addMovieToList("Viking", userId);
+        tut.addMovieToList("Viking", userId);
+
+        // then
+        assertThat(tut.getList(userId)).hasSize(4);
+        assertThat(tut.getList(userId)).extracting(Movie::getTitle).containsExactlyInAnyOrder("Parasite", "Star Wars", "Viking", "Viking");
```

Comment:
Ten sam błąd co w którymś teście wcześniej miałeś. Jak zwracam Ci na coś uwagę to poprawiaj to później w każdym miejscu gdzie robiłeś tak samo